tokio/tests/time_pause.rs

```diff
@@ -215,6 +215,40 @@ async fn interval() {
     assert_pending!(poll_next(&mut i));
 }
 
+#[tokio::test]
+async fn test_time_advance_sub_ms() {
+    time::pause();
+    let now = Instant::now();
+
+    let dur = Duration::from_micros(51_592);
+    time::advance(dur).await;
+
+    assert_eq!(now.elapsed(), dur);
+
+    let now = Instant::now();
+    let dur = Duration::from_micros(1);
+    time::advance(dur).await;
+
+    assert_eq!(now.elapsed(), dur);
+}
+
+#[tokio::test]
+async fn test_time_advance_3ms_and_change() {
+    time::pause();
```

Comment:
```suggestion
#[tokio::test(start_paused = true)]
async fn test_time_advance_3ms_and_change() {
```
Small cleanliness nit